ts/Dashboard/Globals.ts

```diff
@@ -69,6 +72,8 @@ namespace Globals {
         large: 'large'
     };
 
+    export const dashboards: Array<Dashboard|undefined> = [];
```

Comment:
If we allow `undefined` for destroyed Dashboards, we need to have a destroy function setting the value at the index to `void 0`.